packages/governance/src/paramGovernance/paramManager.js

```diff
@@ -137,34 +157,34 @@ const makeParamManagerBuilder = zoe => {
       Nat(v);
       return true;
     };
-    buildCopyParam(name, value, assertNat, ParamType.NAT);
+    buildCopyParam(name, value, assertNat, ParamTypes.Nat);
     return builder;
   };
 
   /** @type {(name: string, value: Ratio, builder: ParamManagerBuilder) => ParamManagerBuilder} */
-  const addRatio = (name, value, builder) => {
-    buildCopyParam(name, value, assertIsRatio, ParamType.RATIO);
+  const addRatioValue = (name, value, builder) => {
```

Comment:
It qualifies what part can be changed. Before it was `BrandedRatio` but _all ratios_ are branded. (same goes for`Amount`) So that was confusing (I can point you to chats).

If you change an `Amount` param, it should change the `Amount` which includes the brand. Are you on board with that?

Then we need a name for the param type in which you can only change the `value` field of the Amount. Thus, `AmountValue` (and `RatioValue`). Is there a better name you'd request?